README.md

```diff
@@ -100,6 +100,22 @@ console.log(some_date_phase);
 }
 ```
 
+### To search for phases between two dates:
+
+```javascript
+// print all full moons in the first quarter of 2014
+var lune = require('lune');
+var phase_list = lune.phase_range(new Date(2014, 1, 1), new Date(2014, 3, 1), 'full')
```

Comment:
Actually, any thoughts on exporting that array, perhaps as `lune.PHASE_NAMES`? I don't know what this buys us unless somebody wants to iterate over them, but it's maybe handy enough to justify itself.